e2e/src/editing_outside_jinja.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { assertThat } from 'hamjest';
+import { activateAndWait, getCustomDocUri, getMainEditorText, getPreviewText, replaceText, setTestContent } from './helper';
+
+suite('Editing outside jinja without recompilation', () => {
+  const DOC_URI = getCustomDocUri('completion-jinja/models/join_ref.sql');
+
+  test('Should not compile outside jinja if error exists', async () => {
+    await activateAndWait(DOC_URI);
+    const initialContent = getMainEditorText();
+    const initialPreview = getPreviewText();
+
+    await replaceText('  )', '   ');
+    await replaceText('select u.id', 'select u.i');
+    assertThat(getPreviewText(), getMainEditorText());
+
+    await setTestContent(initialContent);
+    assertThat(getPreviewText(), initialPreview);
+  });
+});
```

Comment:
Recompilation did not start. But anyway when I change `await replaceText('select u.id', 'select u.i');` I receive different error. Is it right?

Reply:
After this change the next part of code is triggered:
```
async onCompilationFinished(compiledSql: string): Promise<void> {
    if (this.hasDbtError) {
      this.hasDbtError = false;
      this.onGlobalDbtErrorFixedEmitter.fire();
    }
```

Reply:
Could you please clarify what should I change to check this?

Reply:
Now global dbt error could be resolved after any change of non-jinja expression.

Reply:
Fixed it, please check.